modules/luci-mod-dsl/htdocs/luci-static/resources/view/status/dsl/dsl-spectrum.js

```diff
@@ -0,0 +1,85 @@
+//
+// OpenWRT Status menu option to display DSL spectrum graph
+//
+//
+//
+'use strict';
+'require view';
+'require fs';
+'require ui';
+
+
+return view.extend({
+        load: function() {
+
+                  return Promise.all([
+                      fs.exec_direct('/sbin/dsl_cpe_pipe.sh', ['g997dsnrg', '0 1']),
```

Comment:
Just to make sure since I'm not familiar with `dsl_cpe_pipe.sh` - are the `0 1` really meant to be passed as *one* single argument or should it have been `['g997dsnrg', '0', '1']` ?

Reply:
@jow- new never seen an issue with this. `dsl_cpe_pipe.sh` would  throw an error in case of missing arguments.